packages/schematics/angular/application/schema.json

```diff
@@ -107,7 +107,7 @@
       "x-user-analytics": 15
     },
     "strict": {
-      "description": "Creates an application with stricter build optimization options.",
+      "description": "Creates an application with stricter bundle budgets settings.",
```

Comment:
does this not include stricter type checking?

Reply:
The stricter type checking is at workspace level.